tests/executable/test space.tex

```diff
@@ -0,0 +1,4 @@
+\documentclass{article}
```

Comment:
The `the_letter_a` test case is just about the simplest TeX document you can write when using the plain format:

```
a\bye
```

You should be able to use that as the contents of this file if you make sure to pass the argument `--format=plain.fmt.gz` to the `tectonic` executable.

This won't reliably reuse the format generated in-tree, but I'm OK with that for now.